src/prelude.rs

```diff
@@ -1,12 +1,5 @@
+#[doc(no_inline)]
 pub use crate::{
-    document::DocumentOps,
-    group::GroupOps,
-    internal::{
-        document_api::{DocumentId, DocumentName},
-        group_api::{GroupId, GroupName},
-        user_api::{DeviceId, DeviceName, UserId},
-    },
-    policy::PolicyGrant,
-    user::UserOps,
-    DeviceContext, IronOxide, IronOxideErr,
+    common::*, config::*, document::advanced::*, document::*, group::*, policy::*, user::*,
```

Comment:
I was debating between wildcard exports and individual exports, but this will be more maintainable was we add new things